src/styles.css

```diff
@@ -1175,6 +1178,118 @@ textarea {
     width: 1em;
     fill: #fff; }
 
+/**
+ * autocomplete.scss
+ * Define autocomplete input + suggestion styles.
+ * @see: https://github.com/moroshko/react-autosuggest#theme-optional
+ *
+ * @copyright Copyright (c) 2017 Palantir.net
```

Comment:
Update date.

Reply:
Thanks, updated in https://github.com/palantirnet/federated-search-react/pull/26/commits/85fcd96a25e5fde0b32e8a17f4b4813bc222e25f